<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0" columnstretch="0,0">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <layout class="QVBoxLayout" name="_plot_bottom_left_layout"/>
     </item>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="_plot_top_left_layout"/>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="_inputLayout">
       <item>
        <widget class="QRadioButton" name="_estimated_postion_button">
         <property name="text">
          <string>Position estimate</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="_id_anchor_button">
         <property name="text">
          <string>Anchor identification</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="_plot_bottom_right_layout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
